boskos/janitor/janitor.py

```diff
@@ -20,32 +20,33 @@
 import collections
 import datetime
 import json
+import os
 import subprocess
 import sys
 
 
 # A resource that need to be cleared.
-Resource = collections.namedtuple('Resource', 'name group condition managed')
+Resource = collections.namedtuple('Resource', 'name group condition managed tolerate')
 DEMOLISH_ORDER = [
```

Comment:
I think this needs a scarier comment :joy:

Reply:
CAPITALIZED AUTHOR'S WARNING